src/App.js

```diff
@@ -1,36 +1,27 @@
 import React from 'react';
+import TodoList from './TodoList.js';
+import AddToDoForm from './AddTodoForm.js';
 
-const todoList = [
-  {
-    id: '1',
-    title: 'watch lectures'
-  },
-  {
-    id: '2',
-    title: 'read textbook'
-  },
-  {
-    id: '3',
-    title: 'complete assignment'
-  },
 
 
-];
-
 function App() {
+  
+  /* const [newTodo, setNewTodo] = React.useState('');  */
```

Comment:
Go ahead and remove unused code.  If you need it later you can find it in previous commits.